portfolio/src/main/webapp/script.js

```diff
@@ -15,18 +15,31 @@
 /**
  * Adds a random fact to the page.
  */
-function addRandomFact() {
-  const facts =
+
+ const facts =
       ['If I was stuck on a desert island, I would bring a good book', 
       'My favorite animal is a lizard', 'I grew up in Boston, MA', 
       'The food I eat most is Cheerios', 'When I graduate, I plan on moving to Australia', 
       'I learned to surf when I was eight', 'I was the cross country captain for my high school team', 
       'My favorite color is yellow', 'I am super interested in the ocean and marine life'];
+function addRandomFact() {
+  var i = facts.length-1;
```

Comment:
nit: I would suggest naming your variables with something more meaningful.  `i` here for example could be factsLength, factsCount, or factsSize.  

Also probably don't need to hold on to `i` separately.  Something like this would work as well:
`if(facts.length > 0){`